scripts/update_config.js

```diff
@@ -0,0 +1,137 @@
+/*
+ * LiskHQ/lisk-scripts/updateConfig.js
+ * Copyright (C) 2017 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+const fs = require('fs');
+const readline = require('readline');
+const program = require('commander');
+const extend = require('extend');
+const lisk = require('lisk-js');
+
+program
+	.version('0.1.1')
+	.option('-o, --old <path>', 'Old config.json')
```

Comment:
I still think the path to the old config.json should be an arg, not an option. It's not optional. Maybe the new path would also make sense as an (optional) arg?